### PR TITLE
feat: add events page & improve the Download page

### PR DESCRIPTION
**Pull request by qier222**
## Changes:

- feat: add events page
- refactor: use docusaurus.config.js to disable dark mode
- refactor: move users showcase data to a single file
- fix: remove buttons.js
- fix: remove users showcase page

## Screenshots of the change:

4541613391316_ pic

### CODE REVIEW

**Comment by juzhiyuan** [comment]
https://stackoverflow.com/questions/6102297/how-do-i-clean-up-my-github-fork-so-i-can-make-clean-pull-requests

Please refer to this issue to clean PR's commits.

**Review by juzhiyuan — APPROVED** [review]

**Comment by qier222** [comment]
Screenshot of the redesigned download page

CleanShot 2021-02-17 at 15 13 38@2x

File: website/src/pages/downloads/ProjectCard.js [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+          </Button>
+          <Dropdown
+            isDropdownOpen={isDropdownOpen}
+            setIsDropdownOpen={setIsDropdownOpen}
+          >
+            <DropdownItem href={`${downloadLink}.tgz`} target="_blank">
+              Source
+            </DropdownItem>
+            <DropdownItem href={`${downloadLink}.tgz.asc`} target="_blank">
+              ASC
```
Comment (juzhiyuan) [inline website/src/pages/downloads/ProjectCard.js:114]: Hi @qier222, only the Source link needs a mirror, both ASC and SHA512 should use link like `https://downloads.apache.org/apisix/dashboard/2.4/apache-apisix-dashboard-2.4-src.tgz.asc`

Comment (juzhiyuan) [reply website/src/pages/downloads/ProjectCard.js:114]: Please recheck those links from `https://apisix.apache.org/downloads`

File: website/src/pages/downloads/ProjectCard.js [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      });
+    });
+  }, []);
+
+  return (
+    <Card>
+      <LeftSide>
+        <Title>
+          <ShapeBeforeTitle color={color}>{shapeComponent}</ShapeBeforeTitle>
+          {name}
```
Comment (juzhiyuan) [inline website/src/pages/downloads/ProjectCard.js:67]: BTW, not sure if we need to set a hyperlink on the `name` field, so users could visit project's site by clicking name. What's your opinion?

Comment (qier222) [reply website/src/pages/downloads/ProjectCard.js:67]: Great idea, link to GitHub repo?

Comment (juzhiyuan) [reply website/src/pages/downloads/ProjectCard.js:67]: ya

**Review by LiteSun — APPROVED** [review]

**Pull request merged**